CONTRIBUTING.md

```diff
@@ -12,7 +12,7 @@ Here are the basic steps to submit a pull request.
 
 1. Fork the [repo] and clone your forked repo locally on your machine.
 
-1. Follow [setup guidelines] to get the project setup locally.
+1. Follow [setup guidelines](https://github.com/rubyforgood/casa#setup-to-develop) to get the project setup locally.
```

Comment:
Oops! Did I accidentally remove the link from links section below that this could reference? If so, could you look at how we use `[code of conduct]` and `[issues]` to remain consistent?